sdf/bezier.go

```diff
@@ -458,7 +468,15 @@ func (b *Bezier) Polygon() (*Polygon, error) {
 			continue
 		}
 		// Add the spline vertices
-		s.Sample(p, 0, 1, s.f0(0), s.f0(1), 0)
+		f00, err := s.f0(0)
+		if err != nil {
+			return nil, err
+		}
+		f01, err := s.f0(1)
+		if err != nil {
+			return nil, err
+		}
+		s.Sample(p, 0, 1, f00, f01, 0)
 		if i != n-1 {
```

Comment:
So that's a good example of the sort of ugliness that happens when you add error returns to the polynomial evaluation. 1 line becomes 9.

Reply:
yep

Reply:
- reverted this one in 9d211c8